igor-web/src/main/groovy/com/netflix/spinnaker/igor/scm/github/client/GitHubMaster.groovy

```diff
@@ -75,4 +76,12 @@ class GitHubMaster extends AbstractScmMaster {
     }
   }
 
+  @Override
+  Commit getCommitDetails(String projectKey, String repositorySlug, String sha) {
+    try {
+      return gitHubClient.commitInfo(projectKey, repositorySlug, sha)
+    }catch (Exception e) {
+      throw e
+    }
```

Comment:
Since this is just rethrowing any exceptions, it could be simplified to:
```suggestion
    return gitHubClient.commitInfo(projectKey, repositorySlug, sha)
```

Reply:
fixed!